README.md

```diff
@@ -43,7 +43,7 @@ Please see our [user guide](https://zcash.readthedocs.io/en/latest/rtd_pages/rtd
 * :blue_book: See the documentation at the [ReadTheDocs](https://zcash.readthedocs.io)
   for help and more information.
 * :incoming_envelope: Ask for help on the [Zcash](https://forum.z.cash/) forum.
-* :speech_balloon: Chat with our support community on [Discord](https://discordapp.com/channels/669694001464737815/671029188353851393/)
+* :speech_balloon: Join our community on [Discord](https://discordapp.com/invite/PhJY6Pm)
```

Comment:
Discord invites can be time-limited or permanent, which is this one?

Reply:
rex4539Today at 9:19 AM
https://discordapp.com/invite/PhJY6Pm
Hey, you created this invite link. Is it time-limited or permanent?
And I assume it's not user limited.
mineZcashToday at 3:44 PM
It's permanent, it needs to be for the website link